code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Food-Reagents.dm

```diff
@@ -196,42 +194,6 @@ datum/reagent/consumable/condensedcapsaicin/on_mob_life(var/mob/living/M as mob)
 	..()
 	return
 
-datum/reagent/consumable/frostoil
```

Comment:
I would really like Frost Oil to stay as it is. New Cyostylane is fine on its own but its not frost oil. Cyostylane means you cant add cold sauce to foods and the Frost oil you can extract from slimes is less valuable because you need to get oxygen for it to react with. Just leave Frost Oil out of chemistry but leave it alone elsewhere.